common4j/src/main/com/microsoft/identity/common/java/constants/FidoConstants.kt

```diff
@@ -36,5 +36,33 @@ class FidoConstants {
          * Extra query parameter value to declare WebAuthn capability for a host app.
          */
         const val WEBAUTHN_QUERY_PARAMETER_VALUE = "1"
+
+        /**
+         * Header name to signal that the custom passkey protocol should be used.
+         */
+        const val PASSKEY_PROTOCOL_HEADER = "x-ms-PassKeyAuth"
```

Comment:
```suggestion
        const val PASSKEY_PROTOCOL_HEADER_NAME = "x-ms-PassKeyAuth"
```

Reply:
I guess the way you have it is fine as well. But I generally would xx_NAME and xx_VALUE to denote consts for each of those

Reply:
Done: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2181/commits/8a90970d55796d97a386628c3bd025d82e46f23f